content/getting-started/linux.md

```diff
@@ -122,6 +122,10 @@ sudo apt-get update
 sudo apt-get install clang-8 llvm-8-dev lld-8 libclang-8-dev
 ```
 
+For Arch Linux, install the [llvm](https://www.archlinux.org/packages/extra/x86_64/llvm/)
+and [ldd](https://www.archlinux.org/packages/extra/x86_64/ldd/) packages.
```

Comment:
Reading it again, you only mention llvm and lld. Clang is also a dependency, or will it be installed along with one of those packages?

Reply:
Great question, I thought it was a dependency but I might just list it explicitly. Will update!

Reply:
Updated